src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -27,10 +39,19 @@ export const PAGE_SIZE: number = 15
 
 const Conversation: React.FC<ConversationProps> = props => {
   const { conversation, relay, showDetails, setShowDetails } = props
+  const { user } = useSystemContext()
 
   const bottomOfPage = useRef(null)
   const initialMount = useRef(true)
 
+  const isMakeOfferArtwork =
+    conversation.items?.[0]?.item?.__typename === "Artwork" &&
+    conversation.items?.[0]?.item?.isOfferableFromInquiry
+
+  const showBuyerGuaranteeMessage =
+    // @ts-expect-error STRICT_NULL_CHECK
+    userHasLabFeature(user, "Web Inquiry Checkout") && isMakeOfferArtwork
```

Comment:
Ahh, not sure i was clear in my previous comment. We should really strive to make sure that type issues are never ignored and that these flags are never included in PRs with new code. If a type error is encountered, someone on the team can help solve. 

`// @ts-expect-error STRICT_NULL_CHECK` in particular should never be used. That was only added via an automatic codemod to help us migrate to strict type checking. On the rare occasion where something needs to be ignore we can use `@ts-ignore`. 

Whats the type error here? If it's something about the `user` being null, we can add `!` to the end of `user` to tell TS that one is certain the user is not null. But that makes code unsafe. Its better to properly guard: 

```ts
if (user && userHasLabFeature(user, "Web Inquiry Checkout") && isMakeOfferArtwork) {
  ... 
}
```

Reply:
Ah okay. I understand. Thanks for explaining.

Reply:
np! I noticed another dev add a few `// @ts-expect-error STRICT_NULL_CHECK` comments as well so I should probably drop a new message in #dev, I bet a lot of folks didn't see Justins message.